### PR TITLE
add supporting of Complex Image Task

### PR DESCRIPTION
**Pull request by itigertechno**

### CODE REVIEW

File: client/src/Timeouts.php [hunk 1]
```diff
@@ -50,6 +50,12 @@ class Timeouts {
         "timeout" => 1000 * 80
     ];
 
+    public static $complexImageTimeouts = [
```
Comment (azeriker) [inline client/src/Timeouts.php:53]: should be same as imageToTextTimeouts

**Comment by azeriker** [comment]
up useragent version

File: client/src/Timeouts.php (hunk 1 above)
Comment (GANFI) [reply client/src/Timeouts.php:53]: changed

Comment (itigertechno) [reply client/src/Timeouts.php:53]: changed

**Review by azeriker — APPROVED** [review]

**Pull request merged**